docs/releases/2.1.txt

```diff
@@ -221,6 +221,10 @@ Models
 * :meth:`.QuerySet.order_by` and :meth:`distinct(*fields) <.QuerySet.distinct>`
   now support using field transforms.
 
+* :class:`~django.db.models.BooleanField` can now be ``null=True``. This is
+  encouraged instead of :class:`~django.db.models.NullBooleanField`, which will
+  likely be deprecated in the future.
```

Comment:
Fair enough - I understand being cautious with model fields.

Perhaps we could beef up the "future deprecation warning" with admonition blocks in the documentation, and some pointer in the release notes so that users are more aware of the change? I feel that this would encourage change earlier, even before the deprecation warning comes into effect.

Reply:
The documentation for NullBooleanField is two sentences (one of them saying it's not recommended) so I'm not sure that a big warning adds much benefit. I also considered removing the documentation entirely, but it may have some value for someone unfamiliar with Django looking at an existing project.